_issues/hawk.md

```diff
@@ -22,6 +22,14 @@ Representative Richard Hanna (R-NY):
 And this was Rep. Kevin McCarthy's (R-CA) comment: 
 > Everybody thought Hillary Clinton was unbeatable, right? But we put together a Benghazi special committee, a select committee. What are her numbers today? Her numbers are dropping. Why? Because she's untrustable. But no one would have known any of that had happened had we not fought.
 
+<<<<<<< HEAD
+* Sometimes military action is inevitable...and justified. For example, **Hillary supported the raid that killed Osama bin Laden**. However, she chastised GOP Presidential nominees like Ted Cruz for their reckless and cavalier talk, such as ["carpet bombing" ISIS](http://www.npr.org/2016/05/17/478361999/4-things-to-know-about-hillary-clintons-approach-to-foreign-policy). Propsals like carpet bombing don't sound tough, they sound like you don't have a real plan. Hillary knows that slogans are not a strategy, and loose cannons can be dangerous, which is why she has been a smart and steady leader in her role as Secretary of State. 
```

Comment:
This corresponds to the two paragraphs between the `=====` and `>>>> DevProgress/master` markers. Use the format of the paragraphs below but without the **Fact** market.